adopt-a-dog/styling/stylesheet.css

```diff
@@ -0,0 +1,392 @@
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body{
+  background-color: #e6e6e6;
+  width : 100%;
+  max-width: 1450px;
+  min-width: 500px;
+}
+/****** Header ******/
+.page-banner
+{
+  background-image: url("../images/christopher-harris-57366-unsplash.jpg");
+  background-size: 100% 500px;
+  background-repeat: no-repeat;
+  background-position: top;
+  color: white;
+  height: 300px;
+  width: 100%;
+  padding: 10px;
+  display : flex;
+  align-items :center;
+}
+.page-banner-title{
+  padding-left: 30%;
+}
+.page-banner-title h1{
+  color: #4d9900;
+  font-size: 1.5rem;
+  position: absolute;
+  left: 8%;
+  top: 150px;
+  }
+.logo-top{
+ width: 30%;
+ height: 15%;
+ max-width: 35rem;
+ min-width: 15rem;
+ min-height: 10rem;
+ position: absolute;
+ left: 100px;
+ top: 20px;
+}
+.shopping-icon
+{
+ width: 5%;
+ min-width: 0.8rem;
+ height: 1.5rem;
+ position: absolute;
+ top: 30px;
+ right: 180px;
+}
+.banner-price{
+ font-size: 1rem;
+ color: white;
+ position: absolute;
+ top: 35px;
+ right: 120px;
+}
+.checkout-button-top
+{
+ background-color: #66cc00;
+ outline: none;
+ border: none;
+ color: white;
+ font-size: 1rem;
+ width: 12%;
+ min-width: 6rem;
+ padding: 5px;
+ margin: 10px;
+ position: absolute;
+ top: 55px;
+ right: 80px;
+}
+/* Banner Navigation*/
+.nav-bar-container{
+  width : 100%;
+  padding-left: 15%;
+}
+.nav-bar{
+  padding: 0.5rem;
+  width: 40%;
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  list-style-type: none;
+}
+.nav-bar li::after{
+  padding-left: 20px;
+  padding-right: 20px;
+  content: "/";
+}
+.nav-bar li:last-child::after{
+  content: "";
+}
+.nav-bar li a{
+  text-decoration : none;
+  color: #003366;
+  font-size: 1.2rem;
+  font-weight: bold;
+}
+.wrapper{
+  background-color: white;
+  width: 100%;
+  max-width: 1450px;
+  min-width: 500px;
+  padding-left: 15%;
+  padding-right: 15%;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+.page-title{
+  font-size: 2rem;
+  color: #003366;
+}
+/* article = Introduction*/
+.hugging-img{
```

Comment:
maybe a percentage of something? It won't grow at all...

Reply:
These images are going to be the same width no matter the width of the browser window. You'll want them to grow and shrink with the browser becoming narrower. Kind of like what i did as a demo in class.

Reply:
![image](https://user-images.githubusercontent.com/8359784/52538125-a4976000-2d23-11e9-837b-46264a756451.png)

You'll see that the dog image is overflowing the tile.

Reply:
I  changed the image width into %. Kindly let me know whether I did it correctly.

Reply:
Looks great! Make sure you're styling the big intro image correctly though. it doesn't look quite like what we are doing in the pdf.

Reply:
I am made changes in CSS for dog images.